make isDebouncerActive actually return a bool

jakemac53 opened the pull request:
Previously it would return a `truthy` value, but not a boolean as the type annotation specifies.

gronke commented:
Hey @jakemac53, have you seen #2174 ? Looks similar.
https://github.com/gronke/polymer/blob/master/src/micro/debouncer.html#L52

jakemac53 commented:
@gronke yep that also looks like it would solve the issue. I will leave this open for now, but feel free to close this if that one gets merged.

gronke commented on .gitignore:
```diff
@@ -1,3 +1,4 @@
 bower_components
 node_modules
 dist
+.idea
```
what's this line good for?

gronke commented:
I'm in favor of merging this (smaller) PR first, so that the bug is fixed sooner than later.
Will rebase #2174 afterwards, so that it stays fast-forward mergeable.

Except the comment on the `.gitignore` file :+1:

jakemac53 commented:
The gitignore change is for people who use an intellij ide (webstorm, etc).

gronke commented:
@jakemac53 Ah, I see.. Sounds like you want to have this line in your global `.gitignore`, so that it does not have to be duplicated in every project you work on.

jakemac53 commented:
Updated and squashed

JeremybellEU commented:
The changes in this PR already landed in master as of #2174 so this PR can be closed.

kevinpschaaf commented:
@jakemac53 Let's merge this, would you mind rebasing the branch to master? Thanks!

jakemac53 commented:
It looks like the change already got submitted, although you still may want to merge the tests I added so make sure it doesn't regress. I just rebased and pushed so feel free to merge (or not).

kevinpschaaf commented:
@jakemac53 Ah sorry, missed the comments above. Thanks, we'll add the tests.

kevinpschaaf commented:
Tested based on https://github.com/Polymer/polymer/pull/3374 to confirm test failure reported here is fixed.

LTGM
